test/support/model_case.ex

```diff
@@ -30,30 +30,4 @@ defmodule Yggdrasil.ModelCase do
 
     :ok
   end
-
-  @doc """
-  Helper for returning list of errors in model when passed certain data.
-
-  ## Examples
-
-  Given a User model that lists `:name` as a required field and validates
-  `:password` to be safe, it would return:
-
-      iex> errors_on(%User{}, %{password: "password"})
-      [password: "is unsafe", name: "is blank"]
-
-  You could then write your assertion like:
-
-      assert {:password, "is unsafe"} in errors_on(%User{}, %{password: "password"})
-
-  You can also create the changeset manually and retrieve the errors
-  field directly:
-
-      iex> changeset = User.changeset(%User{}, password: "password")
-      iex> {:password, "is unsafe"} in changeset.errors
-      true
-  """
-  def errors_on(model, data) do
-    model.__struct__.changeset(model, data).errors
-  end
```

Comment:
Primarily because it's not used and keeps getting reported on test coverage. That alone is not a good reason I suppose, but for example we use create_changeset on the User model, so calling this would also not work because it assumes `changeset`

and on the game/character stuff just didn't quite figure out how I wanted to use it. Can be resurrected of course but seems limiting with the fact that it assume "changeset" and we already have one case where we don't have that.